test/test_linalg.py

```diff
@@ -2476,28 +2476,6 @@ def test_svd_memory_allocation(self, device, dtype):
         result = torch.linalg.svd(a, full_matrices=False)
         self.assertEqual(result.S, S)
 
-    # This test doesn't work with MAGMA backend https://github.com/pytorch/pytorch/issues/72106
-    @skipMeta
-    @skipCUDAIfRocm
-    @skipCUDAIfNoCusolver
-    @skipCPUIfNoLapack
-    @dtypes(*floating_and_complex_types())
-    def test_svd_nan_error(self, device, dtype):
```

Comment:
Is there something we can do (a documentation update or a comment in the code or OpInfo?) that will spare future developers adding this test again?

Reply:
Merge this PR https://github.com/pytorch/pytorch/pull/88313 :P